cmd/edenStatus.go

```diff
@@ -28,29 +28,53 @@ var statusCmd = &cobra.Command{
 		return nil
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		eveUUID := viper.GetString("eve.uuid")
```

Comment:
Please make this status change into a separate PR and submit to Issue https://github.com/lf-edge/eden/issues/41